### PR TITLE
azure-cli 2.0.25 release notes

### PR DESCRIPTION
**Pull request by derekbekoe**
cc: @sptramer @JasonRShaver

### CODE REVIEW

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.25/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-587) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-587) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C17%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201801171929595562-587%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=4%2Bx6hQKpx0gb046sn8%2B6XpF3S9eFnBh%2BWBkkXIqasg4%3D&st=2018-01-17T19%3A27%3A22Z&se=2018-02-17T19%3A32%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 1]
```diff
@@ -4,18 +4,114 @@ description: Learn about the latest updates to Azure CLI 2.0
...
+
+* Changed `batch login` to return authentication details
+
+### Cloud
+
+* Changed to not require endpoints when setting `--profile` on a cloud
+
+### Consumption
+
+* Added new commands for reservations [X]
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Requires the names of the new commands.

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 2]
```diff
@@ -4,18 +4,114 @@ description: Learn about the latest updates to Azure CLI 2.0
...
+### IoT
+
+* Added support for device provisioning service
+* Added deprecation messages in commands and command help
+* Added IoT check to inform users of the IoT Extension
+
+### Monitor
+
+* Added multi-diagnostic setting support. The `--name` parameter is now required for `monitor diagnostic-settings create`
+* Added command to get diagnostic settings category [X]
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Requires the name of the new command

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 3]
```diff
@@ -4,18 +4,114 @@ description: Learn about the latest updates to Azure CLI 2.0
...
+* Fixed issue when trying to change to/from active-standby mode with `vnet-gateway update`
+* Added support for HTTP2 to `application-gateway [create|update]`
+
+### Profile
+
+* Added support for login with user assigned identities
+
+### Role
+
+* Added `--assignee-object-id` argument [X]
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: No indication of which commands this argument was added to

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 4]
```diff
@@ -4,18 +4,114 @@ description: Learn about the latest updates to Azure CLI 2.0
...
+### Service Fabric
+
+* Added detailed errors to validation response when creating cluster
+* Fixed missing client issue with several commands
+
+### VM
+
+* [PREVIEW] Cross-zone support for `vmss`
+* [BREAKING CHANGE] Changed single-zone `vmss` default to "Standard" load balancer
+* vm/vmss: use right term of "userAssignedIdentity" for EMSI [X]
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Not sure what this means or if it is even a user-facing change

File: docs-ref-conceptual/release-notes-azure-cli.md [hunk 5]
```diff
@@ -4,18 +4,114 @@ description: Learn about the latest updates to Azure CLI 2.0
...
+* [PREVIEW] Added support for OS disk swap
+* Added support for using VM images from other subscriptions
+* vmss: ensure app-gateway has a name when defaults to it for large scalesets [X]
+* `vm/vmss create`: expose `plan` arguments for using custom images with billing informations [X]
```
Comment (sptramer) [inline docs-ref-conceptual/release-notes-azure-cli.md]: Requires information on:

* Which argument(s) were added
* Which commands they were added to

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.25/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-587) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-587) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C17%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201801172103547542-587%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=9Vwp%2FNnCJbmHUz6cF4f%2BVPIEWZtShN2Hu13g2V%2B5hNk%3D&st=2018-01-17T21%3A01%3A24Z&se=2018-02-17T21%3A06%3A24Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs-ref-conceptual/release-notes-azure-cli.md (hunk 1 above)
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

File: docs-ref-conceptual/release-notes-azure-cli.md (hunk 2 above)
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

File: docs-ref-conceptual/release-notes-azure-cli.md (hunk 3 above)
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

File: docs-ref-conceptual/release-notes-azure-cli.md (hunk 4 above)
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: I think this one would make sense to those familiar with EMSI.
Code change is https://github.com/Azure/azure-cli/commit/a60b6197e4510afeb2c8b42920e67fe7cd0171be#diff-88ae33396f6b319fd52598d61bb92b11R79

File: docs-ref-conceptual/release-notes-azure-cli.md (hunk 5 above)
Comment (derekbekoe) [reply docs-ref-conceptual/release-notes-azure-cli.md]: addressed.

**Comment by derekbekoe** [comment]
@sptramer I've pushed new changes from feedback.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.25/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-587) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-587) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C18%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201801180008464497-587%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=5mpxSTg%2FWPBWQwlEPmtUOtC90kVFstRH9MyKG76UVj8%3D&st=2018-01-18T00%3A06%3A48Z&se=2018-02-18T00%3A11%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.25/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-587) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-587) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C18%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201801180133361008-587%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=ANxNKdXiJg5G3244siy%2BfPQLsFmQ17hoZlHpjsExp2c%3D&st=2018-01-18T01%3A31%3A29Z&se=2018-02-18T01%3A36%3A29Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by formulahendry** [comment]
### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs-ref-conceptual/release-notes-azure-cli.md](https://github.com/derekbekoe/azure-docs-cli-python/blob/az-2.0.25/docs-ref-conceptual/release-notes-azure-cli.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-latest&branch=pr-en-us-587) (azure-cli-latest)<br/> [View](https://review.docs.microsoft.com/en-us/cli/azure/release-notes-azure-cli?view=azure-cli-2017-03-09-profile&branch=pr-en-us-587) (azure-cli-2017-03-09-profile) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C1%5C18%5C0c435b1f-40ec-55c2-6a92-4ae06d82d844%5CPullRequest%5C201801180140392586-587%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=NBaHsMr89yt%2FZwa7uzWZ2aWSTveB4EUJeKOPB5pvMow%3D&st=2018-01-18T01%3A38%3A09Z&se=2018-02-18T01%3A43%3A09Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**